cli/stack_template.go

```diff
@@ -95,6 +95,12 @@ func getCloudStorageType(stringFinder func(string) string) cloud.CloudStorageTyp
 		return cloud.GCS
 	case "s3":
 		return cloud.S3
+	case "S3":
+		return cloud.S3
+	case "abfs":
+		return cloud.ABFS
+	case "ABFS":
```

Comment:
it's lowercase only, so this branch is dead

Reply:
@keyki fixed